### ISSUE TITLE
Styling Suggested Values Prior To Selection

### ISSUE THREAD

**Issue by waterboy550** [yairEO/tagify#534]
transformTag works great for styling a tag after it is selected from a whitelist of suggestions. However, I would like to style the suggestions as they appear in the drop down based on the tagData.color values associated with them. Is this possible? See the image from the examples page, if I wanted to color the font red if the text value in the suggestions contained a # in the drop down how would I do that?

![Annotation 2020-06-03 011223](https://user-images.githubusercontent.com/6484336/83598417-ac41d000-a537-11ea-9182-493a79ab0197.png)

**Comment by yairEO** [comment]
See `dropdownItem` in https://github.com/yairEO/tagify#dom-templates

**Comment by waterboy550** [comment]
For those looking for a little more guidance. Simply add a class to the whitelist tagify object, then define the class in the style sheet or style definitions on the page. The dropdownItem function applies the class defined to the drop down item. 

For example, see the below code. I added to the whitelist the class value of redTagDrop or greenTagDrop based on the background color I would like the drop down item to have. Then the following was added to the css style sheet.
`.tags-look .redTagDrop {
                        background-color:#ffe6e6;
                    }
                    .tags-look .greenTagDrop {
                        background-color:#eaffe6;
                    }`

I also have the following for the drop down defined in the code for the tagify object.

`dropdown: {
                            maxItems: 30,
                            classname: "tags-look",
                            enabled: 1,
                            closeOnSelect: true
                        }`
